.atomist/handlers/command/InstallWebHooks.ts

```diff
@@ -144,17 +144,29 @@ class WebHookErrorHandler implements HandleResponse<any> {
     public handle( @ParseJson response: Response<any>): CommandPlan {
         const errors = response.body.errors;
         try {
-            if (errors[0].message === "Hook already exists on this organization") {
+            if (errors != null && errors.length > 0) {
+                if (errors[0].message === "Hook already exists on this organization") {
+                    return CommandPlan.ofMessage(
+                        renderSuccess(`Webhook already installed for ${this.owner} (${this.url})`));
```

Comment:
Most of the time we mention the owner and repo we make it a link and style it owner/repo. I think this is a good convention I'd suggest we do everywhere. The only trick now is getting the GitHub provider URL.